versions/3.0.md

```diff
@@ -395,7 +396,12 @@ Field Pattern | Type | Description
 <a name="parametersDefinitionsObject"></a> | [Parameters Definitions Object](#parametersDefinitionsObject) | An object to hold parameters to be reused across operations. Parameter definitions can be referenced to the ones defined here.
 <a name="responseHeadersDefinitionsObject"></a> | [Response Headers Definitions Object](#responseHeadersDefinitionsObject) | Response headers to reuse across the specification.
 <a name="securityDefinitionsObject"></a> | [Security Definitions Object](#securityDefinitionsObject) | Security definitions to reuse across the specification.
+<a name="linksDefinitionObject"></a> | [Link Definitions Object](#linksDefinitionsObject) | Link definitions to reuse across the specification.
+<a name="callbacksDefinitionObject"></a> | [Callback Definitions Object](#callbacksDefinitionsObject) | Callback definitions to reuse across the specification.
 
+All the fixed fields declared above are objects that MUST contain keys that only use the following characters: `a-zA-Z0-9.-_\`.
```

Comment:
Start the regex with - otherwise this implies all characters between. and _

`-a-zA-Z0-9._\`

Also, is \ intentional or were you trying to escape some other character? This actually renders as if *backslash-space* is in the set of characters, which some my interpret as allowing a space???